pkg/monitor/metrics/application.go

```diff
@@ -116,4 +116,87 @@ var (
+	ClusterMasterNumberGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:        "cluster_master_node_number",
+		Help:        "cluster master node number.",
+		ConstLabels: prometheus.Labels{},
+	}, []string{"cluster"})
+
+	// ClusterMemoryCapacityGauge report the number of MemoryCapacity in cluster
+	ClusterMemoryCapacityGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:        "cluster_memory_capacity",
+		Help:        "cluster memory capacity number.",
+		ConstLabels: prometheus.Labels{},
+	}, []string{"cluster"})
+
+	// ClusterCPUCapacityGauge report the number of CPUCapacity in cluster
+	ClusterCPUCapacityGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:        "cluster_cpu_capacity",
+		Help:        "cluster cpu capacity number.",
+		ConstLabels: prometheus.Labels{},
+	}, []string{"cluster"})
+
+	// ClusterPodCapacityGauge report the number of PodCapacity in cluster
+	ClusterPodCapacityGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:        "cluster_pod_capacity",
+		Help:        "cluster pod capacity number.",
+		ConstLabels: prometheus.Labels{},
+	}, []string{"cluster"})
+
+	// ClusterMemoryAllocatableGauge report the number of MemoryAllocatable in cluster
+	ClusterMemoryAllocatableGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:        "cluster_memory_allocatable",
+		Help:        "cluster memory allocatable number.",
+		ConstLabels: prometheus.Labels{},
+	}, []string{"cluster"})
+
+	// ClusterCPUAllocatableGauge report the number of CPUAllocatable in cluster
+	ClusterCPUAllocatableGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:        "cluster_cpu_allocatable",
+		Help:        "cluster cpu allocatable number.",
+		ConstLabels: prometheus.Labels{},
+	}, []string{"cluster"})
+
+	// ClusterPodAllocatableGauge report the number of PodAllocatable in cluster
+	ClusterPodAllocatableGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:        "cluster_pod_allocatable",
+		Help:        "cluster pod allocatable number.",
+		ConstLabels: prometheus.Labels{},
+	}, []string{"cluster"})
+
+	// ClusterMemoryUsageGauge report the number of MemoryUsage in cluster
+	ClusterMemoryUsageGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:        "cluster_memory_usage",
+		Help:        "cluster memory usage number.",
+		ConstLabels: prometheus.Labels{},
+	}, []string{"cluster"})
+
+	// ClusterCPUUsageGauge report the number of CPUUsage in cluster
+	ClusterCPUUsageGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:        "cluster_cpu_usage",
+		Help:        "cluster cpu usage number.",
+		ConstLabels: prometheus.Labels{},
+	}, []string{"cluster"})
+
+	// ClusterPodUsageGauge report the number of PodUsage in cluster
+	ClusterPodUsageGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:        "cluster_pod_usage",
+		Help:        "cluster pod usage number.",
+		ConstLabels: prometheus.Labels{},
+	}, []string{"cluster"})
```

Comment:
I think these metrics are related to cluster, not directly to applications. I would recommend to move these metrics into a new file under montitor/metrics, like montitor/metrics/multicluster.go.